index.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <link rel="stylesheet" href="/dist/tutor.css" />
+    <script src="http://cdn.mathjax.org/mathjax/2.3-latest/MathJax.js?config=MML_HTMLorMML"></script>
```

Comment:
While discussing this, let me recommend http://stackoverflow.com/a/3832530 just in case MathJax's CDN is offline.